fe1-web/src/core/network/validation/Validator.ts

```diff
@@ -25,7 +25,8 @@ ajv.addSchema([
   ...querySchema,
 ]);
 
-const schemaPrefix = 'https://raw.githubusercontent.com/dedis/popstellar/master/protocol';
+const schemaPrefix =
+  'https://raw.githubusercontent.com/dedis/popstellar/work-be2-dtavaresa-push-gossip/protocol';
```

Comment:
You shouldn't need to change the name of the branch. Feel free to check out PR #1462  from last year if you want to see which files need to change when you add new messages.